iOS/MMKVDemo/MMKVMacDemo/ViewController.mm

```diff
@@ -61,7 +61,10 @@ - (void)funcionalTest {
 
 	[mmkv setDouble:std::numeric_limits<double>::max() forKey:@"double"];
 	NSLog(@"double:%f", [mmkv getDoubleForKey:@"double"]);
-
+    
+    [mmkv setObject:@"hello, mmkv" forKey:@"string"];
```

Comment:
align code please

Reply:
Weird, xcode display 4 indent, but push to the git repo, seems it will reindent use the clang format.

Reply:
Run clang-format on modified files with the `.clang-format` file defined in MMKV's root dir. It'll do the job for you.

Reply:
Thx, formatted